treeherder/client/thclient/client.py

```diff
@@ -734,6 +735,27 @@ def _post_json(self, project, endpoint, oauth_key, oauth_secret, jsondata,
                              timeout=timeout)
         resp.raise_for_status()
 
+    def get_option_collection_hash(self):
+        """
+        Gets option collection hash, a mapping of hash values to build properties
+
+        Returns a dictionary with the following structure:
+
+            {
+                hashkey1: [ { key: value }, { key: value }, ... ],
+                hashkey2: [ { key: value }, { key: value }, ... ],
+                ...
+            }
+        """
+        resp = requests.get('{0}://{1}/api/optioncollectionhash'.format(
+            self.protocol, self.host), timeout=self.timeout)
+        resp.raise_for_status()
+        ret = {}
+        for hash in resp.json():
```

Comment:
Could you give a name other than 'hash' since it is a built-in function? Not necessary but will be good to have.